### PR TITLE
test: ensure params are not deleted from global options

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This is expected to fail at this moment because of #2029.  When the change in https://github.com/googleapis/nodejs-googleapis-common/pull/285 lands, and a new version of the module ships, this should start to pass 😎

### CODE REVIEW

File: test/test.options.ts [hunk 1]
```diff
@@ -99,17 +99,19 @@ describe('Options', () => {
   });
 
   it('should use the same per-API setting twice', async () => {
```
Comment (bcoe) [inline test/test.options.ts:101]: Can you think of any edge cases we might be missing a test for, e.g., we're happy with our tests for per-request options.

Comment (JustinBeckwith) [reply test/test.options.ts]: I think we're in pretty ok shape here.  You can always have more tests, but we test a pretty broad range.
